miden/src/cli/bundle.rs

```diff
@@ -21,6 +24,10 @@ pub struct BundleCmd {
     /// Version of the library, defaults to `0.1.0`.
     #[clap(short, long, default_value = "0.1.0")]
     version: String,
+    /// Build a kernel library from module `kernel` and using the library `dir` as kernel
+    /// namespace.
```

Comment:
```suggestion
    /// Build a kernel library from module `kernel` and using the library `dir` as kernel
    /// namespace. The `kernel` file should not be in the directory `dir`.
```

A comment is probably fine for now, and hopefully #1436 fixes this properly

Reply:
done